File: pset1/mario/less/mario.c
#include <stdio.h>
#include <cs50.h>

void hashtags(int k);

int main(void)
{
    // Get the desired height
    int height;
    do
    {
        height = get_int("Width: ");
    }
    while (height < 1 || height > 8);

    // Loop thru height
    for (int i = 0; i < height; i++)
    {
        // Print height minus linenumber amount of spaces
        for (int j = 0; j < height - i - 1; j++)
        {
            printf(" ");
        }

        hashtags(i);
        printf("\n");
    }
}

// Print right amount of hashtags according to line number
void hashtags(int k)
{
    for (int j = 0; j < k + 1; j++)
    {
        printf("#");
    }
}

File: pset6/mario/less/mario.py
# Import get_int from cs50
from cs50 import get_int

# Check if height is between 1 and 8. Else repeat
while True:
    height = get_int("Height: ")
    if height >= 1 and height <= 8:
        break

# Print spaces and hashtags "the python way"
for i in range(1, height + 1):
    print(" " * (height - i) + "#" * i)